styles/_base.scss

```diff
@@ -18,35 +18,80 @@ body {
   }
 
   > main {
```

Comment:
Can we not style tags directly.

Reply:
A lot of this is the ZenDesk code and not something I've written. 
I won't refactor it all but I'll see what I can do.